packages/atomic/package.json

```diff
@@ -36,8 +36,11 @@
     "test:watch": "stencil test --spec --watchAll",
     "generate": "stencil generate",
     "cypress:test": "cypress run --record --key 0e9d8bcc-a33a-4562-8604-c04e7bed0c7e --browser chrome",
-    "cypress:test:firefox": "cypress --record --key 0e9d8bcc-a33a-4562-8604-c04e7bed0c7e --browser firefox",
+    "cypress:test:firefox": "cypress run --record --key 0e9d8bcc-a33a-4562-8604-c04e7bed0c7e --browser firefox",
     "cypress:open": "cypress open",
+    "cypress-hsp:test": "cypress run --config-file cypress-hsp.json --browser chrome",
+    "cypress-hsp:test:firefox": "cypress run --config-file cypress-hsp.json --browser firefox",
```

Comment:
Why firefox too?

Reply:
Since Cypress supports Firefox and this comes at no extra cost, I was thinking we might want to have our tests run again both FF and the default Chrome-based test. I had added the Firefox command to the package.json a while back for the regular tests, but didn't follow up about integrating them into our pipelines.